blazor/datagrid/editing.md

```diff
@@ -1083,4 +1083,67 @@ In the following code example, the Employee Name is a foreign key column. When e
         public string FirstName { get; set; }
     }
 }
+```
+
+## Enum column editing
+
+You can edit the enum type data in the grid column using the Edit Template feature of the Grid.
+
+In the following sample, the `SfDropDownList` component is rendered in the [EditTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_EditTemplate) for the Employee Feedback column. The enumerated list data can be bound to the Employee Feedback column using the two-way binding (**@bind-Value**).
```

Comment:
In the following sample, the `SfDropDownList` component is rendered in the [EditTemplate]: